packages/plugin-core/src/test/suite-integ/LookupJournal.test.ts

```diff
@@ -46,6 +44,14 @@ suite("Scratch Notes", function () {
           });
           const newNote = getNoteFromTextEditor();
           expect(newNote.fname.startsWith(`${fname}.journal`)).toBeTruthy();
+          // The note title should be in the format yyyy-MM-dd
+          expect(/\d{4}-\d{2}-\d{2}$/g.test(newNote.title)).toBeTruthy();
+
+          // @ts-ignore
+          const traits = newNote.traitIds;
```

Comment:
i've done this too but in general, not a fan of having `@ts-ignore` in tests. does it make sense to establish a convention where `ts-ignore` code needs to be encapsulated in a file?

- test/suite-integ/ts-ignore-utils.ts
```tsx
getNoteTrait(note: NoteProps) [
//@ts-ignore
return note.traitIds
}
```

in the future, we want to enable strict typechecks and dis-allow `@ts-ignore` in the code base unless its in a file called `ts-ignore-*`

Reply:
I bypassed the `ts-ignore` with a cast + a TODO comment.

> Does it make sense to establish a convention where ts-ignore code needs to be encapsulated in a file?

IMO - probably not worth it; I think it'll make things slightly more convoluted (and it may also lead to more circular deps).  Instead, let's add a point in our PR guidelines + coding conventions to not allow adding of additional `ts-ignore`'s going forward.  And then let's eliminate existing `ts-ignore`'s until we get down to 0, and then we can strengthen the ts compiler settings.

I can take a look at the docs and amend.